compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/ProjectUtils.java

```diff
@@ -636,6 +647,15 @@ private static void addDependencyContent(StringBuilder content, String org, Stri
         return possiblePkgNames;
     }
 
+    private static boolean isBuiltInPackage(ImportModuleRequest importModuleRequest) {
```

Comment:
We have a isBuiltInPackage in package descriptor can we make both use a single API

Reply:
Added this separate method since in  L636 we don't have access to `PackageDescriptor`.

Reply:
then can package description use the Util function ?